[Fabric-Sync] Simplify and update the fabric sync guide (#34993)
project-chip/connectedhomeip: docs/guides/fabric_synchronization_guide.md

pairing already-discovered 3 20202021 <ip> 5540
```

After the device is successfully added, you will observe the following on
Ecosystem 2 with the newly assigned Node ID:

```
>>> New device with Node ID: 0x3 has been successfully added.
```

> Additionally, you should also get notified an new device is added on the

Reviewer: ```suggestion
Additionally, you should also get notified when a new device is added to
```

Ecosystem 2 from the Ecosystem 1:

```

> >>> A new devie is added on Endpoint 3.

Reviewer 2: nit
```suggestion
>>> A new device is added on Endpoint 3.
```
